pkgs/development/python-modules/pyenchant/default.nix

```diff
@@ -13,14 +13,20 @@ buildPythonPackage rec {
     sha256 = "fc31cda72ace001da8fe5d42f11c26e514a91fa8c70468739216ddd8de64e2a0";
   };
 
-  propagatedBuildInputs = [ pkgs.enchant1 ];
+  propagatedBuildInputs = [ enchant2 ];
 
   patchPhase = let
```

Comment:
This should be done in `postPatch` so that it does not break `patches` attribute.

Reply:
Do you mean that instead of 'patchPhase' this should be 'postPatch'?

Reply:
Yup. The default [`patchPhase`](https://github.com/NixOS/nixpkgs/blob/3a4e550944cc0cf760766111944dd24f13081be6/pkgs/stdenv/generic/setup.sh#L917-L943) does some things so replacing it is not a good idea.

Reply:
Done in the latest patchset; Also realized I could consolidate the substituteInPlaces invocations into a single invocation.